webapp/javascript/components/FileList.tsx

```diff
@@ -29,17 +30,23 @@ const getBodyRows = (
       (profId) => profId === profile.id
     );
 
+    const date = parseISO(profile.updatedAt);
+    const timeString = `${format(date, 'MMM d, yyyy')} at ${format(
+      date,
+      'H:mm'
+    )}`;
```

Comment:
I think we are missing `PM/AM` there
![image](https://user-images.githubusercontent.com/6951209/194092784-9cd1700d-9c7f-4304-ac03-0279985523de.png)

Reply:
Yeah, I like the 24h but it's definitely confusing, specially when it's alone like this. Plus we can say we have a large US audience.

Reply:
I like 24h format I think this is fine